iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -413,4 +413,10 @@ public void forceUpdateRegionPriority(
   public void removeRegionRouteCache(TConsensusGroupId regionGroupId) {
     loadCache.removeRegionRouteCache(regionGroupId);
   }
+
+  /** Force balance the region leader. */
+  public void forceBalanceRegionLeader() {
+    routeBalancer.balanceRegionLeader();
+    routeBalancer.balanceRegionPriority();
```

Comment:
Please broadcast a RouteChangeEvent here, for which u can reference from StatisticsService.updateLoadStatistics()